src/main/java/com/sumologic/log4j/SumoLogicAppender.java

```diff
@@ -118,6 +116,7 @@ public long cost(String e) {
     }
 
     @PluginFactory
+    @Deprecated
```

Comment:
When deprecating a method, a good idea is to point to what to use instead. Either in the annotation, or in a comment above. Otherwise it's kind of a frustrating experience for users of the "library".